go.mod

```diff
@@ -5,6 +5,7 @@ go 1.16
 require (
 	github.com/Microsoft/go-winio v0.4.17
 	github.com/Microsoft/hcsshim v0.8.16
+	github.com/Microsoft/hcsshim/test v0.0.0-20210227013316-43a75bb4edd3
```

Comment:
Can we please avoid `github.com/Microsoft/hcsshim/test` in the main go.mod?

Reply:
Do we have a good place for that? We have https://github.com/containerd/containerd/tree/master/integration/client but the tests in the PR are not in the integration/ folder.